orchestration/transfer_controller.py

```diff
@@ -115,18 +63,29 @@ def copy(
 
 class GlobusTransferController(TransferController[GlobusEndpoint]):
     """
-    Use Globus Transfer to move data between endpoints.
+    Use Globus Transfer to move data between Globus endpoints.
+
+    This controller handles the transfer of files between Globus endpoints using the
+    Globus Transfer API. It manages authentication, transfer submissions, and status tracking.
 
     Args:
-        TransferController: Abstract class for transferring data.
+        config (BeamlineConfig): Configuration object containing Globus endpoints and credentials
     """
+
     def __init__(
         self,
-        config: Config832,
+        config: BeamlineConfig,
         prometheus_metrics: Optional[PrometheusMetrics] = None
     ) -> None:
         super().__init__(config)
         self.prometheus_metrics = prometheus_metrics
+        logger.debug(f"Initialized GlobusTransferController for beamline {config.beamline_id}")
```

Comment:
Since you're calling the `super().__init__`, I would assume you're getting this log twice. Since the super is logging yhr class, this is extraneous, no?